react/src/views/Box/components/HistoryEntries.tsx

```diff
@@ -0,0 +1,60 @@
+import { ListItem, ListIcon, List, Stack, Flex } from "@chakra-ui/react";
+import { IconType } from "react-icons";
+import { MdCheckCircle, MdSettings, MdHistory } from "react-icons/md";
+import { HistoryEntry } from "types/generated/graphql";
+
+interface IHistoryEntriesProps {
+  data: HistoryEntry[];
+  total: number | undefined;
+}
+function HistoryEntries({ data, total }: IHistoryEntriesProps) {
+  // sort history entries in the descending order
```

Comment:
I'll create a backend PR to have the entries sorted by most recent first already, then the logic can be removed here.

Reply:
@vahidbazzaz I might directly push the change to this branch, already tested it locally

Reply:
@pylipp sure please push the change, that should be fine